docs/supported_ops.md

```diff
@@ -424,9 +424,9 @@ Accelerator supports are described below.
 <td>S</td>
 <td><b>NS</b></td>
 <td><b>NS</b></td>
-<td><b>NS</b></td>
-<td><b>NS</b></td>
-<td><em>PS<br/>unionByName will not optionally impute nulls for missing struct fields  when the column is a struct and there are non-overlapping fields;<br/>max nested DECIMAL precision of 18;<br/>UTC is only supported TZ for nested TIMESTAMP;<br/>missing nested BINARY, CALENDAR, ARRAY, UDT</em></td>
+<td><em>PS<br/>max nested DECIMAL precision of 18;<br/>UTC is only supported TZ for nested TIMESTAMP;<br/>missing nested BINARY, CALENDAR, ARRAY, UDT</em></td>
```

Comment:
Looks like we don't support arrays of arrays? Is that what you expected?

Reply:
Array support split off into another branch for simplicity and because of nested type support concerns (need to test+support maps of arrays and structs containing arrays or else nested arrays can't be supported)